### ISSUE TITLE
Feature request - PID Controller Blocks and Tools.

### ISSUE THREAD

**Issue by JorgeGMarques** [rafaelaroca/BIPES_ui_testing#20]
Working on that, coming soon.

**Comment by JorgeGMarques** [comment]
Just implemented the basics in c3f9447 using my [m-lundberg/simple-pid ](https://github.com/m-lundberg/simple-pid) fork/port from Python to MicroPython available at [JorgeGMarques/micropython-simple-pid](https://github.com/JorgeGMarques/micropython-simple-pid) (documentation available [here](https://micropython-simple-pid.readthedocs.io/)).

I'll probably normalize the gains in a next commit because if we set `Ki = 1` at `scale ='ms'` equals `Ki = 1000` at `scale ='s'` because:
```
now = self.time()
dt = utime.ticks_diff(now,self._last_time)
self._integral += self.Ki * error * dt
```
where `self.time()` is defined by `scale`, e.g. `self.time() = utime.ticks_ms()` at  `scale ='ms'`.

**Comment by JorgeGMarques** [comment]
With commit b54b138, turnings are coherent between scales.
